### PR TITLE
adjust year

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @DeepDiver1975 

fixes https://github.com/owncloud/core/issues/13469

### CODE REVIEW

File: templates/wizard.php [hunk 1]
```diff
@@ -63,7 +63,7 @@
 	and <a href="http://owncloud.org/promote"
 		target="_blank">spread the word</a>!')); ?>
 <?php else: ?>
-© 2014 <a href="https://owncloud.com" target="_blank">ownCloud Inc.</a>
+© 2015 <a href="https://owncloud.com" target="_blank">ownCloud Inc.</a>
```
Comment (DeepDiver1975) [inline templates/wizard.php]: @karlitschek shouldn't that be a period like 2012-2015 - just from a legal pov? THX

Comment (karlitschek) [inline templates/wizard.php]: From a legal perspective it doesn´t make a difference. Maybe 2012-2015 looks nicer

Comment (DeepDiver1975) [inline templates/wizard.php]: or is it 2010-2015 as we celebrated ownCloud's 5th birthday?

Comment (MorrisJobke) [inline templates/wizard.php]: Done :)

**Comment by DeepDiver1975** [comment]
@MorrisJobke I'm sorry - it's 2011-2015 - see https://owncloud.com/privacy-legal/

**Comment by MorrisJobke** [comment]
@DeepDiver1975 Fixed

**Comment by DeepDiver1975** [comment]
thx

**Pull request merged**